aiohttp/web.py

```diff
@@ -388,7 +389,9 @@ def run_app(app: Union[Application, Awaitable[Application]], *,
             reuse_address: Optional[bool]=None,
             reuse_port: Optional[bool]=None) -> None:
     """Run an app locally"""
-    loop = asyncio.get_event_loop()
+    loop = asyncio.new_event_loop()
```

Comment:
Why create a new one?

Reply:
I've asked @NewUserHa to make `run_app` as close to `asyncio.run()` as possible.
Please look

Reply:
Alright, so perhaps a comment in the code explaining the motivation would be helpful